### PR TITLE
chore: more comments in injectForOrigin function

### LINKED ISSUE #1251: [proposal] empty cpuset string should be ignored in reconciler of koordlet runtime hooks
**What is your proposal**:
image

empty string is illegal for cpuset cgroup

**Why is this needed**:

**Is there a suggested solution, if so, please add it**:

### PR DESCRIPTION
**Pull request by Tiana2018**
### Ⅰ. Describe what this PR does

add some comments in injectForOrigin Function

### Ⅱ. Does this pull request fix one issue?

fixes #1251 

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/runtimehooks/protocol/container_context.go [hunk 1]
```diff
@@ -190,7 +190,9 @@ func (c *ContainerContext) ReconcilerDone(executor resourceexecutor.ResourceUpda
 	c.injectForOrigin()
 }
 
+// Inject illegal parameters
```
Comment (saintube) [inline pkg/koordlet/runtimehooks/protocol/container_context.go:193]: This comment should be removed. The parameters below are legal.

File: pkg/koordlet/runtimehooks/protocol/protocol_test.go [hunk 2]
```diff
@@ -17,8 +17,11 @@ limitations under the License.
 package protocol
 
 import (
+	"sync"
 	"testing"
 
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/resourceexecutor"
```
Comment (saintube) [inline pkg/koordlet/runtimehooks/protocol/protocol_test.go:23]: please move the private pkg at the last

File: pkg/koordlet/runtimehooks/protocol/protocol_test.go [hunk 3]
```diff
@@ -156,3 +176,85 @@ func TestPodResponse_ProxyDone(t *testing.T) {
 		})
 	}
 }
+func TestPodResponse_ReconcilerDone(t *testing.T) {
```
Comment (saintube) [inline pkg/koordlet/runtimehooks/protocol/protocol_test.go:179]: please left a blank line between functions

**Review by saintube — COMMENTED** [review]
@Tiana2018 The CI fails. Please check the [DCO](https://github.com/koordinator-sh/koordinator/pull/1264/checks?check_run_id=13232452297) and sign-off the commit.

> DCO
> There is one commit incorrectly signed off. This means that the author of this commit failed to include a Signed-off-by line in the commit message.
> To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).

File: pkg/koordlet/runtimehooks/protocol/protocol_test.go (hunk 3 above)
Comment (Tiana2018) [reply pkg/koordlet/runtimehooks/protocol/protocol_test.go:179]: Thank you for your suggestion.

**Comment by saintube** [comment]
/lgtm
PTAL @zwzhang0107

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**